Source/LightsOutSln/LightsOut/Patches/ModCompatibility/Biotech/PatchGeneAssembly.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using LightsOut.Common;
+using LightsOut.Patches.Power;
+using RimWorld;
+using Verse;
+
+namespace LightsOut.Patches.ModCompatibility.Biotech
+{
+    public class PatchGeneAssembly : ICompatibilityPatchComponent<PatchGeneAssembly>
+    {
+        public override string ComponentName => "Patch for gene assembly related things";
+        private const string PROCESSOR_ID = "GeneProcessor";
```

Comment:
The other constants in this mod use pascal case, and this name isn't very descriptive.

I'd rename it something like `GeneProcessorDefName`.